### ISSUE TITLE
Look for and delete empty directories when deleting files from torrent

### ISSUE THREAD

**Issue by jfurrow** [jfurrow/flood#399]
In https://github.com/jfurrow/flood/issues/158#issuecomment-310873594, a user wrote:
> Is it possible to delete the empty directories left over from the torrents?

**Comment by bbashy** [comment]
I'd like to help fix this as most torrents would contain the files within a folder and these aren't being deleted.

Ref: https://github.com/Novik/ruTorrent/blob/master/plugins/erasedata/update.php

**Comment by potatoattack** [comment]
I don't think this is an issue with flood, more of a rtorrent configuration issue.

`method.set_key = event.download.erased, remove_file,"execute={rm,-drf,--,$d.data_path=}"`

in your rc file will remove any directories (and other files in the directory; the contents of a .rar, for example) when you delete a torrent from flood, make sure you have the `d.data_path` command defined.

**Comment by zawapete** [comment]
@potatoattack rutorrent supports this without any need for a specific rtorrent configuration, I would hope for the same in flood

**Comment by T-z3P** [comment]
Hello. I'm having the same issue even with Flood and rTorrent running under the same user. Any update?

**Comment by noraj** [comment]
@T-z3P No, the only developer (jfurrow) is not actively developing flood anymore. You can take a look at #712 and #672

**Comment by Whisper40** [comment]
Hello, this bug has been present for more than a year, is it possible to do something to fix it, this kind of problem seems basic for a torrent client, just like displaying disk space...

**Comment by artik** [comment]
@Whisper40 It's on rtorrent side, just add in your rtorrent.rc:

`method.set_key = event.download.erased,delete_erased,"execute=rm,-rf,--,$d.data_path="`

And Flood will properly delete folder and content.
